src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1919,7 +1883,7 @@ because xbuild doesn't support framework reference assemblies.
     <_ProguardConfiguration Include="$(ProguardConfigFiles)" />
   </ItemGroup>
   <ItemGroup Condition=" '$(ProguardConfigFiles)' == '' ">
-    <_ProguardConfiguration Include="$(_AndroidSdkDirectory)tools\proguard\proguard-android.txt" />
+    <_ProguardConfiguration Include="$(_AndroidSdkDirectory)tools\proguard\proguard-android.txt" Condition="Exists ('$(_AndroidSdkDirectory)tools\proguard\proguard-android.txt')" />
```

Comment:
Had to add this cos of a bunch of test failues. This file is not present on newer Android SDK installations